Convert-AzureVMsetup.ps1

```diff
@@ -0,0 +1,46 @@
+#-------------------------------------------------------------------------------------------------------------
```

Comment:
might be able to do without this script, see my comments on AzureVM.bicep about loading a file as base64